test/performance/roles/nanotube/templates/config.toml.j2

```diff
@@ -24,3 +24,10 @@ PprofPort = 6060
 # Time between termination signal and forced termination of the process. This helps when graceful
 # shutdown is stuck or slow.
 TermTimeoutSec = 1
+
+RateLimiterEnabled = true
```

Comment:
It seems a bit strange to enable rate limiting in the default configuration used for _performance_ testing. It will not be possible to run a typical performance test with the limiter on. While it is necessary to test the current PR, please consider removing it.